homeassistant/components/manual_mqtt/alarm_control_panel.py

```diff
@@ -87,15 +86,18 @@
 
 def _state_validator(config):
     """Validate the state."""
-    config = copy.deepcopy(config)
```

Comment:
This looks exactly the same as the copying happening in the `manual` integration: https://github.com/home-assistant/core/pull/9264/commits/5dc624f57455bd89c5b63369fcc3478fb2896509#r137995580

I think the problem was mutating nested dictionaries, this PR now copies the nested dictionaries instead of mutating them.
However, tests pass also without the copying, so I'm not sure if we still need it?